docs/operator/configuration.md

```diff
@@ -0,0 +1,68 @@
+# Configuration
+
+Learn about the default configuration in Busola and [how to change it](#changing-the-configuration).
```

Comment:
I'd remove this link. The doc is pretty short, so it's easy to find the section.